src/java.base/share/classes/java/lang/template/TemplateProcessor.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package java.lang.template;
+
+import jdk.internal.javac.PreviewFeature;
+
+/**
+ * This interface simplifies declaration of
+ * {@linkplain ValidatingProcessor template processors}
+ * that do not throw checked exceptions. For example:
+ * {@snippet :
+ * TemplateProcessor<String> processor = st -> {
+ *     List<String> fragments = st.fragments();
+ *     List<Object> values = st.values();
+ *     // check or manipulate the fragments and/or values
+ *     ...
+ *     return StringTemplate.interpolate(fragments, values);
+ * };
+ * }
+ *
+ * @param <R>  Processor's process result type.
```

Comment:
Some code block/link missing?

Reply:
`};` is in the snippet. `}` is the end of the snippet.